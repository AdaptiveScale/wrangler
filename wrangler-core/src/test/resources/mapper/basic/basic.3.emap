// This is a test for using functions and also assignments from already assigned values.
out.c = 1l; // time();
out.f = (double) out.c;
out.x = 3.4d;