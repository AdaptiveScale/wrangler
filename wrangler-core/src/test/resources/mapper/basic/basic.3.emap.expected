{"a":null,"b":null,"c":1,"d":null,"e":null,"f":1.0,"x":3.4,"rec2a":null,"rec3":[]}